src/legacy/core_plugins/metrics/public/components/vis_types/gauge/series.test.tsx

```diff
@@ -17,16 +17,16 @@
  * under the License.
  */
 import React from 'react';
+
+// @ts-ignore
```

Comment:
What is the added `ts-ignore` for here?

Reply:
Rolled back to a js file temporarily (until we deangularize vis types)